libunicorn-sys/src/lib.rs

```diff
@@ -96,12 +96,4 @@ impl fmt::Display for Error {
 }
 
 #[cfg(feature = "std")]
-impl std::error::Error for Error {
-    fn description(&self) -> &str {
-        self.msg().as_bytes()
-    }
-
-    fn cause(&self) -> Option<&std::error::Error> {
-        None
-    }
-}
+impl std::error::Error for Error {}
```

Comment:
`description` is soft-deprecated. Users are expected to use Display trait instead.
cause is now deprecated in rust 1.33.0. The `Error::source` which replace it is
is auto return `None`.